ui/app/AppLayouts/Communities/views/EditAirdropView.qml

```diff
@@ -45,6 +45,10 @@ StatusScrollView {
     // Bool property indicating whether the fees are available
     required property bool feesAvailable
 
+    property string enabledChainIds
+    property string networkThatIsNotActive
```

Comment:
It would be better to make it "private" since it's an internal implementation property.